apps/mobile/src/components/Community/CommunityView.tsx

```diff
@@ -24,12 +33,28 @@ export function CommunityView({ queryRef }: Props) {
           }
           ... on Community {
             __typename
-            name
-            description
+            chain
             contractAddress {
               ...LinkableAddressFragment
             }
+            creator {
+              __typename
+              ... on GalleryUser {
+                username
+                universal
+                ...ProfilePictureFragment
+              }
+              ... on ChainAddress {
+                ...LinkableAddressFragment
+              }
+            }
+            tokensInCommunity(first: 10000) {
```

Comment:
does the value for `first` make a difference?

Reply:
That's a good catch @kaitoo1! I set it to `1` since we only need the `total` data.